de.dashup.model/src/main/resources/de/dashup/model/builder/panels/1-large.html

```diff
@@ -1,13 +1 @@
-<dashup-grid-layout id="grid-container" col-size="9 3">
-    <dashup-text-field id="input-city" label="City" value="Walldorf"></dashup-text-field>
-    <dashup-button label="OK"
-                   api="http://api.openweathermap.org/data/2.5/weather?units=metric&appid=524da7907b1939626510c547ae65d67c"
-                   api-param="q:input-city"
-                   consumers="label-temperature label-humidity"></dashup-button>
-
-    <dashup-display id="label-temperature" label="Temperature:" api-key="main.temp" quantity="°C"></dashup-display>
-    <dashup-display></dashup-display>
-    <dashup-display id="label-humidity" label="Humidity:" api-key="main.humidity" quantity="%"></dashup-display>
-    <dashup-display></dashup-display>
-
-</dashup-grid-layout>
\ No newline at end of file
+<dashup-weather></dashup-weather>
```

Comment:
I still do not get why I should do this? This takes us the possibility to add extra logic to our widgets and adds extra DOM content to the light DOM. The other way around, our widgets would be more encapsulated and more powerful (or it would be much easier to add extra features at least). I can change it if you feel better that way, but do not complain if the users are bored from the widgets in the end.

Reply:
We should do this because we should not create more powerful widgets than the user will be able to create. I think the user is way more bored when he sees that there are so powerful widgets and he can just create very basic things. 
In addition to that I think we should handle our widgets the same way as we handle the user widgets so there is one single style and way for all widgets.
This was a result of a team discussion so you cant change that by your own.

Reply:
This wasn't really the result of a team discussion, but rather how you interpreted the result. But never mind, if the team wants it then this is the direction we will go to. What do you think @raphaelmue @svenleonhard ?
I just do not see the benefit for our users, this just restricts the capabilities that dashup could have. Of course, users would not be able to create widgets like we do, but they can open feature requests at any time. Rendering, styling etc. would still be the same for dashup widgets and user widgets.